### PR TITLE
Hook notifications drawer to chrome-services websockets

### PR DESCRIPTION
**Pull request by aneelac22**
[RHCLOUD-25965]

### CODE REVIEW

File: src/components/NotificationsDrawer/DrawerPanelContent.tsx [hunk 1]
```diff
@@ -65,7 +66,17 @@ const DrawerPanelBase = ({ innerRef }: DrawerPanelProps) => {
...
+    try {
+      const notifications = await axios.get('/api/notifications/v1/notifications/drawer');
+      dispatch(populateNotifications(notifications.data.data));
+    } catch (error) {
+      console.error('Unable to get Notifications ', error);
+    }
+  }
+
   useEffect(() => {
+    getNotifications();
```
Comment (Hyperkid123) [inline src/components/NotificationsDrawer/DrawerPanelContent.tsx:79]: The drawer should be populated on the Chrome start-up. Even if the drawer is closed. We need the data to change the toggle state if needed.

Getting the data after the drawer is opened is too late.

**Review by florkbr — APPROVED** [review]
Overall looks good to me for now. I have a few concerns around adding some tests and documentation for the newly added TS types (NotificationsPayload etc.), but that's also an issue for the entire repo (not just this PR). Confirmed with @Hyperkid123 offline that this PR works against the chrome-service WS.

**Comment by florkbr** [comment]
Jenkins is confused by itself again 😮‍💨 (PR check container name conflict)

**Pull request merged**